nimble/controller/src/ble_ll.c

```diff
@@ -1549,6 +1549,10 @@ ble_ll_init(void)
     features |= BLE_LL_FEAT_LE_CODED_PHY;
 #endif
 
+#if (MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_PERIODIC_ADV) == 1)
```

Comment:
I'd rather use `#if MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_PERIODIC_ADV)` since this is common form in code - no idea why we have it written as it is now...

Reply:
OK,  I'll later on also send PR that cleans that up in other places